coder_sniffer/Drupal/Test/Classes/FullyQualifiedNamespaceUnitTest.inc.fixed

```diff
@@ -2,8 +2,66 @@
 
 /**
  * @file
+ * Example.
  */
 
-use Drupal\my_module\Example\MyUrlHelper;
+use Test\Foo;
+use Test\NotUsed;
+use Test\Bar;
+use Test\Alias as TestAlias;
 
-$x = new MyUrlHelper();
+/**
+ * Example.
+ */
+class Example {
+
+  /**
+   * Description.
+   */
+  public function test1(TestAlias $alias) {
+
+  }
+
+  /**
+   * Description.
+   */
+  public function test2(NotUsed $notUsed) {
+
+  }
+
+  /**
+   * Description.
+   */
+  public function test3(TestAlias $alias) {
+
+  }
+
+  /**
+   * Description.
+   */
+  public function test4(Foo $foo) {
+
+  }
+
+  /**
+   * Description.
+   */
+  public function test5(Bar $bar) {
+
+  }
+
+  /**
+   * Description.
+   */
+  public function test6(TestDiff\Bar $bar) {
+
+  }
+
+  /**
+   * Description.
+   */
+  public function test7(TestDiff\TestAlias $anotherAlias) {
```

Comment:
we report an error here, but we do not fix it. I guess we could, but let's leave this out of scope for this fix for now.